context/tasks/todo.md

```
@@ -154,6 +154,70 @@ Reference:
 - `context/tasks/web-and-tui-bugfix.md`
 - `context/tasks/tui-opencode-fork-alignment-plan.md`
 
+### PR: Core Runtime Decomposition Pass
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)